### DISCUSSION TITLE
Perte de connection apres 10mn avec Strongswan Android application

### DISCUSSION THREAD

**Discussion by thir06** [strongswan/strongswan#1643] [Q&A]
Bonjour

Je me permets de poster car apres des heures de recherches,  je ne resouds pas le probleme de perte de connection ou plutot de dialogue entre mon server Strongswan et mon application Strongswan Android sur un telephone.
Pendant 10mn tout fonctionne parfaitement,  mes divers applications (conversations, element, K9mail ..)  se connectent and dialoguent correctement et apres 10mn  je perds la communication et les applications ne se connectent plus.  

Auriez vius deja rencontrer ce genre de probleme svp ?

merci
thierry

**Comment by tobiasbrunner** [comment]
Please stick to English. And please read #196.
